apis_core/apis_entities/models.py

```diff
@@ -778,9 +778,7 @@ def merge_with(self, entities):
         from apis_core.apis_metainfo.models import Uri
 
         e_a = type(self).__name__
-        self_model_class = ContentType.objects.get(
-            app_label="apis_entities", model__iexact=e_a
-        ).model_class()
+        self_model_class = ContentType.objects.get(model__iexact=e_a).model_class()
```

Comment:
Here is a DB call on `ContentType`, and then on the received instance `model_class()`. 

This could be easily replaced by using our caching module like so `caching.get_entity_class_of_name(e_a)`

Reply:
Thanks for the comment. We will keep that in mind, a separate PR with a global introduction of the caching module wherever possible would probably be more appropriate. Another approach is to hide the caching implementation details by simply providing a `get_class` method in the entity.

Reply:
I agree that a global replacement PR would be better. I made an issue: https://github.com/acdh-oeaw/apis-core-rdf/issues/108

And while we could attach a method to a model class, I'm sceptical. What's the benefit of it in contrast to using the caching module? I see it rather as redundancy and minor inconsistency. 

Calling such a function through a class method does not hide more implementation than calling function itself already imo.